Softeq.XToolkit.Permissions.Droid/PermissionsManager.cs

```diff
@@ -75,7 +75,9 @@ private async Task<PermissionStatus> CommonCheckWithRequestAsync<T>()
                 return permissionStatus;
             }
 
-            if (permissionStatus == PermissionStatus.Denied && IsPermissionRequested<T>())
+            if (permissionStatus == PermissionStatus.Denied
+                && IsPermissionDeniedEver<T>()
+                && !Xamarin.Essentials.Permissions.ShouldShowRationale<T>())

```

Comment:
As I understand, `IPermissionService` is our proxy for Essentials.Permissions. Maybe we should move this method into `PermissionService` and extend interface?

Reply:
Currently, [XToolkit.Permissions fully depend on Xamarin.Essentials](https://softeq.github.io/XToolkit.WhiteLabel/articles/xtoolkit/permissions.html) for Services and Managers, contains Interfaces over Essentials and some custom behavior only (double-dialogs).

TODO: merge platform-specific services into one
(the idea is: single service for extend XE, and 2 managers for custom UI behavior)

Reply:
Moved to IPermissionService